app/javascript/packs/home.js

```diff
@@ -2,6 +2,8 @@ import Vue from 'vue';
 import router from '../src/router';
 import store from '../src/store/index';
 import app from '../src/app';
+import VueCookies from 'vue-cookies'
```

Comment:
Expected empty line after import statement not followed by another import  import/newline-after-import<br>Absolute imports should come before relative imports                       import/imports-first<br>Missing semicolon                                                          semi